README.rst

```diff
@@ -71,6 +71,21 @@ Installation
     ./venv.sh
     . venv/bin/activate
 
+File permissions
+----------------
+Newly created key files are only accessible by ``simp_le`` user (mode
+600) while certificates are restricted by the umask setting.
+
+Should you need more relaxed permissions, either create an empty file
+first or modify the permission afterwards. ``simp_le`` will not modify
```

Comment:
I don't see any tests for this behavior (one for empty file, one for persisted mode). Also, I'm not sure If like like the "empty file" solution...